jinseok/app.js

```diff
@@ -94,7 +104,54 @@ app.get('/users/:userId/posts', async function (req, res, next) {
     [userId, userId]
   );
 
-  res.status(200).json(data);
+  res.status(200).json({ data: post });
+});
+
+app.put('/posts', async function (req, res, next) {
+  const { update, userId, postId } = req.body;
+
+  await dataSource.query(
+    `
+  UPDATE posts SET context= ?, updated_at=CURRENT_TIMESTAMP
```

Comment:
```suggestion
  UPDATE posts SET context= ?
```
updated_at 컬럼에 대한 스키마를 작성할 때, ON UPDATE를 사용하였기 때문에 mysql에서 자동으로 현재시간을 넣어줍니다. 따라서 직접 작성하지 않아도 됩니다.